### DISCUSSION TITLE
Unable to render kroki / k8s helm

### DISCUSSION THREAD

**Discussion by stblassitude** [requarks/wiki#2895] [Help / Questions]
I've just set up wiki.js using the helm chart (from the default branch), and many things appear to be working quite nicely; I'm particularly happy about the draw.io integration!

But I'd like to also make use of some of the diagram gererators, in particular kroki. If I insert this into a markdown page:
```
```kroki
rackdiag {
  16U;
  1: UPS [2U];
  3: DB Server;
  4: Web Server;
  5: Web Server;
  6: Web Server;
  7: Load Balancer;
  8: L3 Switch;
}
```

Then I just get a standard code block in both the preview and in the rendered page. In the rendering pipeline, kroki is enabled (default).

If I look at the JS console, I get these error messages:
```
Failed to load resource: the server responded with a status of 404 ()
Refused to execute https://wiki.app.example.com/_assets/js/prism/prism-kroki.min.js as script because "X-Content-Type-Options: nosniff" was given and its Content-Type is not a script MIME type.
```

Is there something I need to adjust, or is the file somehow missing or in a different path in the Docker image?

**Comment by wujun4code** [comment]
same with me.

**Comment by batistein** [comment]
I have the same issue. Is there some workaround?

**Comment by tfboy** [comment]
You might need an extra carriage return for it to work.
This works on my wikijs:
```
```kroki
rackdiag
{
  16U;
  1: UPS [2U];
  3: DB Server;
  4: Web Server;
  5: Web Server;
  6: Web Server;
  7: Load Balancer;
  8: L3 Switch;
}
```
and don't forget the closing backticks!

**Comment by stblassitude** [comment]
Not for me. I've taken a screenshot since embedding Markdown in Markdown is complicated.
<img width="1355" alt="kroki-test" src="https://user-images.githubusercontent.com/202994/226618039-648072cb-a838-4899-b2eb-75a026866e40.png">

**Reply by tfboy** [reply]
No, I mean a carriage return between the rackdiag and the opening {
So it's:
```
rackdiag
{
16U; ...
```
and not
```
rackdiag {
16U; ...
```

**Comment by tfboy** [comment]
This is the same for all rendering done by Kroki.
The format is always:
```
```kroki
<name of diagram type>
{
<code of diagram>
}
```.
